data/tn.json

```diff
@@ -0,0 +1,22 @@
+{
+    "state": "Tennessee",
+    "pages": [
+        {
+            "source": "",
+            "paragraphs": [
+               "\"To incorporate in the state of Tennessee you must follow the following steps:\r\n1. Choose a Corporate Name.\r\n2. Appoint a Registered Agent. \r\n3. Set Up a Corporate Records Book. \r\n4. Prepare Corporate Bylaws. \r\n5. Appoint Initial Corporate Directors. \r\n6. Hold Your First Board of Directors Meeting. \r\n7. Issue Stock. \r\n8. Comply with\u00A0Tennessee\u00A0Annual Report Requirements.\r\n9.  Comply with Tennessee Annual Report Requirements\r\n10.  Comply With Other Tax and Regulatory Requirements\r\n\r\n2. Appoint a Registered Agent. ...\r\n3. Set Up a Corporate Records Book. ...\r\n4. Prepare Corporate Bylaws. ...\r\n5. Appoint Initial Corporate Directors. ...\r\n6. Hold Your First Board of Directors Meeting. ...\r\n7. Issue Stock. ...\r\n8. Comply with\u00A0Tennessee\u00A0Annual Report RequirementS.\r\n9.  Comply with Tennessee Annual Report Requirements\r\n10.  Comply With Other Tax and Regulatory Requirements\"",
```

Comment:
i'm noticing the `\r\n` in these files, can we make sure it only is `\n`. `\r` is an artifact of windows.